Makefile

```diff
@@ -26,7 +26,7 @@ update-file-format:
 	done
 
 .PHONY: fix-meta-schema
-fix-meta-schema:
+fix-meta-schema: compile-schema
```

Comment:
For whatever reason I had the habit of `make all-meta-schema`, which would then skip the yaml -> json step causing all sorts of mayhem.

Probably the generation script can use a safety guard to fail when the JSON is empty